[2025-10-09T04:51:28.438+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: order_generator.load_orders scheduled__2025-10-08T11:00:00+00:00 [queued]>
[2025-10-09T04:51:28.465+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: order_generator.load_orders scheduled__2025-10-08T11:00:00+00:00 [queued]>
[2025-10-09T04:51:28.466+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2025-10-09T04:51:28.472+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2025-10-09T04:51:28.477+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2025-10-09T04:51:28.512+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): load_orders> on 2025-10-08 11:00:00+00:00
[2025-10-09T04:51:28.521+0000] {standard_task_runner.py:55} INFO - Started process 2505 to run task
[2025-10-09T04:51:28.539+0000] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'order_generator', 'load_orders', 'scheduled__2025-10-08T11:00:00+00:00', '--job-id', '140', '--raw', '--subdir', 'DAGS_FOLDER/order_loader_dag.py', '--cfg-path', '/tmp/tmpw2vm0wqq']
[2025-10-09T04:51:28.545+0000] {standard_task_runner.py:83} INFO - Job 140: Subtask load_orders
[2025-10-09T04:51:28.737+0000] {task_command.py:389} INFO - Running <TaskInstance: order_generator.load_orders scheduled__2025-10-08T11:00:00+00:00 [running]> on host 81fbc710bbce
[2025-10-09T04:51:28.889+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=order_generator
AIRFLOW_CTX_TASK_ID=load_orders
AIRFLOW_CTX_EXECUTION_DATE=2025-10-08T11:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-10-08T11:00:00+00:00
[2025-10-09T04:51:28.943+0000] {logging_mixin.py:137} INFO - Successfully loaded data to orders
[2025-10-09T04:51:29.262+0000] {logging_mixin.py:137} INFO - Successfully loaded data to order_items
[2025-10-09T04:51:29.267+0000] {python.py:177} INFO - Done. Returned value was: None
[2025-10-09T04:51:29.286+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=order_generator, task_id=load_orders, execution_date=20251008T110000, start_date=20251009T045128, end_date=20251009T045129
[2025-10-09T04:51:29.361+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2025-10-09T04:51:29.386+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
